src/components/singleCountry.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable */
```

Comment:
- [**Optional**] It's not a good practice of disabling Linters. These are great tools that monitor your codebase and flag programmatic/stylistic errors. It is very important to fix these issues.